Finality Proof of GRANDPA BEEFY Light Client.md

```diff
@@ -0,0 +1,47 @@
+## Original Process of Proof
+
+A ***GRANDPA*** light client needs to verify the validity of a state before accepting a state update. A simple approach is to provide the signatures of the members of the ***GRANDPA*** Authority Set and the supporting evidence along with the state to the light client, so that the light client knows whether there are enough authority members to endorse the state. If more than 2/3 of members sign it(we assume less than 1/3 dishonest Authorities), the light client can accept the state.
+
+## An Improvement Proposal
+
+However, the reality is far from ideal because the number of members of the GRANDPA authority set is so large that 2/3 + 1 signatures and their evidence still account for a large amount of data. Therefore, we need to find ways to reduce this data volume while ensuring sufficient security (the ideal security goal is that the cost of attacking the bridge is equal to the sum of the market value of DOT and ETH).
+
+We can reduce the required number of signatures to reduce the amount of data.
+
+First, we assume that less than 1/3 members in the authority set are dishonest. We can randomly sample 1/3 of the verifier's signatures; the light client can be confident that the state is valid if all the sampled signatures are the same.
```

Comment:
Here 1/3 is the estimated result, and we actually use the selection of K values to control the desired attack cost

Reply:
```suggestion
First, we assume that less than 1/3 members in the authority set are dishonest. We can randomly sample 1/3 of the verifier's signatures; the light client can be confident that the state is valid if all the sampled signatures are the same. Here, 1/3 is the estimated result, and we actually use the selection of K and MIN_SUPPORT values to estimate the desired attack cost.
```